contrib/pyln-spec/setup.py

```diff
@@ -0,0 +1,53 @@
+from setuptools import setup
```

Comment:
I just want to make sure you know *why* the "../setup.py" in the sub- setup.py files works.

It reflects a great deal of, um, opreational creativity?

Reply:
@cdecker i'm deferring this one to you :)